### PR TITLE
Update quantifiers-in-regular-expressions.md

### LINKED ISSUE #19075: Wrong wording in example of * quantifier
I checked carefully, this is the only instance of this mistake on this page:
"The following example illustrates this regular expression. Of the nine digits in the input string, five match the pattern and four (95, 929, 9219, and 9919) do not."

There are way more than nine digits in the input string.  The best word to use instead of 'digits' here is 'numbers', 'decimal integers' would be used by Vulcans and OCD programmers.  'Groups of digits' would also be correct but awkward and unusual.  The word "digit" is consistently used to mean one single decimal digit 0 to 9 everywhere else on this page.  One of the most common hallmarks of bad programming documentation is confusion of "digit" and "decimal integer literal" -- very rarely seen on these pages (except in this one spot).
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7254044f-fe8a-ff27-8e5e-4d01e5c4e58a
* Version Independent ID: a1d9ffdc-d888-68cb-ac05-dd5018c89c9c
* Content: [Quantifiers in Regular Expressions](https://docs.microsoft.com/en-us/dotnet/standard/base-types/quantifiers-in-regular-expressions)
* Content Source: [docs/standard/base-types/quantifiers-in-regular-expressions.md](https://github.com/dotnet/docs/blob/master/docs/standard/base-types/quantifiers-in-regular-expressions.md)
* Product: **dotnet**
* Technology: **dotnet-standard**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by swapniljha001**
## Type of PR

- [x] Documentation changes

## Summary

Changed `digit` to `decimal integers`.

Fixes #19075

### CODE REVIEW

File: docs/standard/base-types/quantifiers-in-regular-expressions.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Quantifiers specify how many instances of a character, group, or character class
 ### Match Zero or More Times: *  
  The `*` quantifier matches the preceding element zero or more times. It is equivalent to the `{0,}` quantifier. `*` is a greedy quantifier whose lazy equivalent is `*?`.  
   
- The following example illustrates this regular expression. Of the nine digits in the input string, five match the pattern and four (`95`, `929`, `9219`, and `9919`) do not.  
+ The following example illustrates this regular expression. Of the nine decimal integers in the input string, five match the pattern and four (`95`, `929`, `9219`, and `9919`) do not.
```
Comment (adegeo) [inline docs/standard/base-types/quantifiers-in-regular-expressions.md:45]: Per the discussion in the original issue, we settled on **digit groups**.

```suggestion
 The following example illustrates this regular expression. Of the nine digit groups in the input string, five match the pattern and four (`95`, `929`, `9219`, and `9919`) do not.  
```

**Review by adegeo — CHANGES_REQUESTED** [review]
Greetings!

Thank you for this contribution. We really appreciate it when community members, such as yourself, step up and help us improve the documentation! One thing to adjust and then I'll accept the PR.

**Comment by swapniljha001** [comment]
Thanks, @adegeo for the kind words. I have made the necessary changes and hopefully, this will close the #19075 issue.

**Review by adegeo — APPROVED** [review]

**Comment by adegeo** [comment]
Thanks again!

**Pull request merged**